release-notes/v1_12.md

```diff
@@ -1,4 +1,4 @@
----
+﻿---
```

Comment:
There may be an encoding issue here (on the file) which seems to be breaking the header